### ISSUE TITLE
yubico-piv-tool terminated - stack smashing detected

### ISSUE THREAD

**Issue by alukyan** [Yubico/yubico-piv-tool#119]
Using `yubico-piv-tool -a status` on Ubuntu running on VirtualBox VM on Windows.
It works but at the end I get a message 

`*** stack smashing detected ***: yubico-piv-tool terminated`

This is probably related to another message `Output buffer to small, wanted to write 2068, max was 2048.Slot 9a`

Full output from the command is as follows:

```
CHUID:	3019d<redacted>fe00
Output buffer to small, wanted to write 2068, max was 2048.Slot 9a:	
	Algorithm:	RSA2048
	Subject DN:	< redacted >
	Issuer DN:	< redacted >
	Fingerprint:	< redacted >
	Not Before:	Jun 29 22:35:53 2017 GMT
	Not After:	Jun 29 22:35:53 2018 GMT
*** stack smashing detected ***: yubico-piv-tool terminated
Aborted (core dumped)
```

**Comment by klali** [comment]
what version of yubico-piv-tool is this with?

**Comment by alukyan** [comment]
yubico-piv-tool 1.0.3

**Comment by klali** [comment]
You'll need to update, this bug was fixed quite some time ago, latest release is 1.4.3.
